cmd/node/config/config.toml

```diff
@@ -104,6 +104,18 @@
         MaxBatchSize = 20000
         MaxOpenFiles = 10
 
+[TxLogsStorage]
+    Enabled = false
```

Comment:
this should not be optional. logs and events should be saved by all validators

Reply:
done